components/footer.jsx

```diff
@@ -4,6 +4,10 @@ import { IntlMixin, FormattedHTMLMessage } from 'react-intl';
 var Footer = React.createClass({
   mixins: [IntlMixin],
   render: function() {
+    var firefox_footer = this.getIntlMessage('firefox_footer');
+    if (window.location.pathname.indexOf('thunderbird') !== -1) {
```

Comment:
What if we had seperate footers that may or may not wrap this one, and pass in the message?

We would still need to go into each thunderbird page and switch it to use the new component, but we wouldn't need to pass in the message on each one. Better?